src/Paramore.Brighter/ControlBusSenderFactory.cs

```diff
@@ -69,6 +69,7 @@ public IAmAControlBusSender Create<T, TTransaction>(IAmAnOutbox outbox,
                 CommandProcessorBuilder.StartNew()
                 .Handlers(new HandlerConfiguration())
                 .DefaultPolicy()
+                .DefaultResilencePipeline()
```

Comment:
I wonder if we have DefaultResilencePipeline() and internally call DefaultPolicy i.e. don't need to have both on the API but need the results for backwards compatibility?